backend/src/main/kotlin/org/loculus/backend/controller/SubmissionControllerDescriptions.kt

```diff
@@ -53,10 +53,10 @@ const val SUBMIT_PROCESSED_DATA_DESCRIPTION = """
 Submit processed data as a stream of NDJSON. The schema is to be understood per line of the NDJSON stream. 
 This endpoint performs validation (type validation, missing/required fields, comparison to reference genome) on the data
 returned by the processing pipeline, so that it can technically be used for release. On a technical error, this endpoint
- will roll back all previously inserted data. It is the responsibility of the processing pipeline to ensure that the 
- content of the data is correct. If the pipeline is unable to provide valid data, it should submit the data with errors.
- In this case, no validation will be performed and the status of the accession version will be set to 'HAS_ERRORS'.
- The user can then edit the data and submit a corrected version.
+will roll back all previously inserted data. It is the responsibility of the processing pipeline to ensure that the 
+content of the data is correct. If the pipeline is unable to provide valid data, it should submit the data with errors.
+In this case, no validation will be performed and the status of the accession version will be set to 'HAS_ERRORS'.
```

Comment:
There are also some internal docs that use those two terms, I added that to the acceptance criteria earlier today https://github.com/loculus-project/loculus/issues/3031#issuecomment-2449304609

Reply:
This particular code area I just saw that the whitespace was messy, so I fixed it. `HAS_ERRORS` still exists as a status on the Pipeline output (would have to look up the exact table) so I didn't replace it here. The change was just done in the view.

I also did a search for both terms and I think I got everything!

Reply:
Thanks - I hadn't understood that HAS_ERRORS existed in two different places. I suspect we may no longer need it. (I.e. it may is presumably redundant with the actual errors themselves, which are I presume what we are using) But (A) I may be wrong (B) it doesn't matter much.

Reply:
I briefly touched on this with Anya and she said it's probably best to just leave the pipeline code as-is for now

Reply:
Sorry think I missed some context here - where does HAS_ERRORS exist in the pipeline output?

Reply:
Some investigation: it's not directly in the pipeline output, it's computed by the backend here: https://github.com/loculus-project/loculus/blob/ee8a842ff9a403cbe062cabd632cd3792974d5f0/backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt#L311

Reply:
Huh, okay! I was always just working on this view, and just saw that the HAS_ERRORS is still there in a different table, so I thought maybe it's used there also for a bunch of things. But if we can get rid of it that would be good I think!

Reply:
Who can assist me with this @chaoran-chen ? I'm not sure but I agree that it kind of looks like we don't actually needs this status at all anymore?

Reply:
In the latest version of this PR, this was removed.